fix: make download button accessible via keyboard

morsko1 opened the pull request:
<!--- Provide a general summary of your changes in the Title above -->
Make download button accessible via keyboard.

### Description
<!--- Describe your changes in detail -->
Download button was implemented using `div` tag. That's why it was inaccessible via keyboard.



### Motivation and Context
<!--- Why is this change required? What problem does it solve? -->
<!--- If it fixes an open issue, please link to the issue here. -->
<!--- Use the magic "Fixes #1234" format, so the issues are -->
<!--- automatically closed when this PR is merged. -->
Fixes #8881 



### How Has This Been Tested?
<!--- Please describe in detail how you manually tested your changes. -->
<!--- Include details of your testing environment, and the tests you ran to -->
<!--- see how your change affects other areas of the code, etc. -->
See description in #8881 


### Screenshots (if appropriate):



## Checklist
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->

### My PR contains... 
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [ ] No code changes (`src/` is unmodified: changes to documentation, CI, metadata, etc.)
- [ ] Dependency changes (any modification to dependencies in `package.json`)
- [x] Bug fixes (non-breaking change which fixes an issue)
- [ ] Improvements (misc. changes to existing features)
- [ ] Features (non-breaking change which adds functionality)

### My changes...
- [ ] are breaking changes to a public API (config options, System API, major UI change, etc).
- [ ] are breaking changes to a private API (Redux, component props, utility functions, etc.).
- [ ] are breaking changes to a developer API (npm script behavior changes, new dev system dependencies, etc).
- [x] are not breaking changes.

### Documentation
- [x] My changes do not require a change to the project documentation.
- [ ] My changes require a change to the project documentation.
- [ ] If yes to above: I have updated the documentation accordingly.

### Automated tests
- [x] My changes can not or do not need to be tested.
- [ ] My changes can and should be tested by unit and/or integration tests.
- [ ] If yes to above: I have added tests to cover my changes.
- [ ] If yes to above: I have taken care to cover edge cases in my tests.
- [ ] All new and existing tests passed.

char0n commented:
@morsko1 you've change the order of `Copy` and `Download` buttons. Any particular reason?

morsko1 commented:
> @morsko1 you've change the order of `Copy` and `Download` buttons. Any particular reason?

@char0n Actually, the button order in the template was incorrect. The `Copy` button is displayed before the `Download` button in the UI (you can see that in Petstore example).

So, I have updated the button order and changed the tag of the `Download` button from `div` to `button`.

As a result, the keyboard navigation works as expected.

char0n reviewed (approved):
LGTM! I can confirm keyboard navigation now works by running the branch locally. Thanks for contributing!
